deploy/upstream/0.1.0/07_operator.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: marketplace-operator
+  namespace: marketplace
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: marketplace-operator
+  template:
+    metadata:
+      labels:
+        name: marketplace-operator
+    spec:
+      serviceAccountName: marketplace-operator
+      containers:
+        - name: marketplace-operator
+          image: quay.io/openshift/origin-operator-marketplace:latest
```

Comment:
If you are saying that the upstream version is `0.1.0` then this would need to be tagged. So we should either remove the version from the directory layout or figure out a way to tag the image.

Reply:
If we don't care specifically about versioning, lets just remove it.

Reply:
I am fine with that for now. But I am guessing we would have to introduce versioning at some point for upstream.

Reply:
Let's cross that bridge when we get there.